app/api/probe_dock/metrics_api.rb

```diff
@@ -44,13 +44,19 @@ def current_organization
           end
         end
 
-        def apply_projects_and_users_filters(rel_for_filtering)
+        def apply_projects_filters(rel_for_filtering)
           rel = rel_for_filtering
 
           if params[:projectIds].present? && params[:projectIds].kind_of?(Array)
             rel = rel.where 'projects.api_id IN (?)', params[:projectIds].collect(&:to_s)
```

Comment:
Method calls with arguments should have parentheses (in several places in this file). We should refactor old code as we add new features.